### PR TITLE
Update team page for 2023

### PR DESCRIPTION
**Pull request by ericholscher**
I think this info is correct, but I did trim it down a bit, so please let me know if there's anything missing! Also, we're missing bios for some folks, so if you're like your bio on there, send it over to me and I'll get it updated.

----
:books: Documentation preview :books:: https://writethedocs-www--1923.org.readthedocs.build/

### CODE REVIEW

**Comment by diggerlilly** [comment]
Somehow, my old name is still in my bio..eek and gasp!
s/Alicia Duncan/Alicia Lewis por favor

or I'm happy to do separate PR for `bios/alicia.rst`..just let me know.

File: docs/conf/portland/2023/team.rst [hunk 1]
```diff
@@ -20,32 +20,60 @@ Samuel Wright
 
 {% include "bios/sam.rst" %}
 
+Mo Nishiyama
+~~~~~~~~~~~~
+
+{% include "bios/mo.rst" %}
+
+Alicia Duncan
```
Comment (plaindocs) [inline docs/conf/portland/2023/team.rst]:
```suggestion
Alicia Lewis
```

Comment (plaindocs) [reply docs/conf/portland/2023/team.rst]: @diggerlilly FYI GitHub also (if you hover over the @ )

**Comment by ericholscher** [comment]
Gonna merge this, but let me know if there's other updates needed!

**Pull request merged**